### PR TITLE
Remove warning messages

### PR DESCRIPTION
**Pull request by martagregula**

### CODE REVIEW

File: frontend/src/components/taskList/taskListRow.tsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ export default function TableRowComponent(props: any) {
   const [open, setOpen] = React.useState(false);
   const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
   const [anchorEl2, setAnchorEl2] = React.useState<null | HTMLElement>(null);
-  const [users, setUsers] = React.useState();
+  const [, setUsers] = React.useState();
```
Comment (LukaszLik) [inline frontend/src/components/taskList/taskListRow.tsx:100]: delete " , "

File: frontend/src/components/taskList/taskListRow.tsx [hunk 2]
```diff
@@ -276,6 +273,7 @@ export default function TableRowComponent(props: any) {
       AuthenticationService.getUsers().then((response) => {
         setUsers(response.data.data);
 
+        // eslint-disable-next-line array-callback-return
```
Comment (LukaszLik) [inline frontend/src/components/taskList/taskListRow.tsx]: "forEach() may be preferable when you’re not trying to change the data in your array, but instead want to just do something with it"
i wtedy pozbyć się // eslint-disable-next-line array-callback-return

File: frontend/src/components/taskList/taskListRow.tsx [hunk 3]
```diff
@@ -285,6 +283,7 @@ export default function TableRowComponent(props: any) {
             });
           }
 
+          // eslint-disable-next-line array-callback-return
```
Comment (LukaszLik) [inline frontend/src/components/taskList/taskListRow.tsx]: same?

File: frontend/src/components/taskList/taskListRow.tsx (hunk 1 above)
Comment (martagregula) [reply frontend/src/components/taskList/taskListRow.tsx:100]: nie bo krzyczy :(

File: frontend/src/components/taskList/taskListRow.tsx (hunk 2 above)
Comment (martagregula) [reply frontend/src/components/taskList/taskListRow.tsx]: done

File: frontend/src/components/taskList/taskListRow.tsx (hunk 3 above)
Comment (martagregula) [reply frontend/src/components/taskList/taskListRow.tsx]: done

**Pull request merged**